deployments/charts/kuma/values.yaml

```diff
@@ -127,6 +127,8 @@ controlPlane:
     pullPolicy: IfNotPresent
     # -- Kuma CP image repository
     repository: "kuma-cp"
+    # -- Kuma CP Image tag
```

Comment:
how about
```
    # -- Kuma CP Image tag. When not specified, the value is copied from global.tag
```

Reply:
make check is giving me some weird errors:
/Users/dasson/bin/protobuf.d/include: warning: directory does not exist.
/Users/dasson/go/pkg/mod/github.com/envoyproxy/protoc-gen-validate@v0.4.1: warning: directory does not exist.
/Users/dasson/go/pkg/mod/github.com/envoyproxy/data-plane-api@: warning: directory does not exist.
/Users/dasson/go/pkg/mod/github.com/cncf/udpa@: warning: directory does not exist.
/Users/dasson/go/pkg/mod/github.com/googleapis/googleapis@: warning: directory does not exist.
validate/validate.proto: File not found.
mesh/v1alpha1/dataplane.proto:12:1: Import "validate/validate.proto" was not found or had errors.
make: *** [protoc/mesh/v1alpha1] Error 1

Tried resolving it, but to no success :(

Reply:
Done. Fixed.